### PR TITLE
Fix: Assignee text box alignment

### LINKED ISSUE #1224: Make the text box appear where the name is on the task details page
### Issue Description

When the superuser opens the task details of a specific developer on the status-site task details page, on this page, they see various information about a specific task, there is a field called Assignee to see who is doing that task, In there if any superuser wants to change the assignee from the developer to any other developer, the assigned text box appearing below the Assignee name filed, but the text box should appear where the Assignee field is.
### Expected Behavior

The text box for changing the assignee should appear directly where the Assignee field is located, not below it.
### Current Behavior

The text box for changing the assignee appears below the Assignee name field instead of appearing where the Assignee field is.
### Screenshots

Achintya_tasks

### Reproducibility

 - [x] This issue is reproducible
 - [ ] This issue is not reproducible

### Steps to Reproduce
1. Open the status-site task details page for any developer on staging and log on as a superuser.
2. Look at the "Assignee" field.
3. Look at the "Assignee" field.
4. Try to change the assignee from the current developer to another developer.
5. Observe where the text box for changing the assignee appears.

### Severity/Priority
 - [ ] Critical
 - [x] High
 - [ ] Medium
 - [ ] Low

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by Achintya-Chatterjee**
Date: 31 July, 2024

Developer Name: @Achintya-Chatterjee 

---

## Issue Ticket Number

- Closes #1224 
## Description
 - Updates the `TaskDetails` component and its corresponding test to ensure the assignee input field appears correctly where the name is in edit mode. Also, remove the feature flag from those stable features for superusers as @ankushdharkar asked me to.

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

https://github.com/user-attachments/assets/6f9b9baf-448b-4cda-ba1a-8830051d9ce3

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

Screenshot 2024-07-31 at 21 51 33

Screenshot 2024-08-04 at 00 11 50

</details>

## Additional Notes
- Ensured that the feature worked as expected after removing the feature flag from different features. Also, it didn't break other features as well. I verified that the assignee input appeared (where the name is) correctly on the task details page in edit mode.

### CODE REVIEW

**Comment by yesyash** [comment]
- can we make the suggestion box width same as the input box?

**Review by yesyash — CHANGES_REQUESTED** [review]

**Comment by yesyash** [comment]
- please remove the duplicate input box shown in the video

File: src/components/taskDetails/index.tsx
Comment (vinit717) [inline src/components/taskDetails/index.tsx:1]: Can you please break this component

File: src/components/taskDetails/index.tsx [hunk 1]
```diff
@@ -367,42 +358,69 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
...
-                                />
+                                <div className={styles.detailsWithInput}>
+                                    <label className={styles.detailType}>
+                                        Assignee:
+                                    </label>
+                                    <div className={styles.inputContainer}>
+                                        {isEditing && isUserAuthorized ? (
+                                            <>
+                                                <input
+                                                    type="text"
```
[the remark this answers is not in the document]
Comment (Achintya-Chatterjee) [reply src/components/taskDetails/index.tsx]: @yesyash added, thanks for the suggestion

**Comment by Achintya-Chatterjee** [comment]
> * can we make the suggestion box width the same as the input box?

@yesyash can you see the video once again ?

File: src/components/taskDetails/index.tsx
Comment (Achintya-Chatterjee) [reply src/components/taskDetails/index.tsx:1]: I request you to review the `TaskDetails` component one more time after review if you think we need to break this component, I'll break it.

**Comment by Achintya-Chatterjee** [comment]
> * please remove the duplicate input box shown in the video

@yesyash removed.

File: src/components/taskDetails/index.tsx [hunk 2]
```diff
@@ -415,37 +433,32 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
...
-                                                    setNewEndOnDate(
-                                                        e.target.value
-                                                    );
-                                                }}
-                                                onBlur={handleBlurOfEndsOn}
-                                                value={newEndOnDate}
-                                                data-testid="endsOnTaskDetails"
-                                            />
-                                        </>
+                                <div className={styles.detailsWithInput}>
```
[the remark this answers is not in the document]
Comment (Achintya-Chatterjee) [reply src/components/taskDetails/index.tsx]: @vinit717 changed the class name

**Comment by vinit717** [comment]
> > * can we make the suggestion box width the same as the input box?
> 
> @yesyash can you see the video once again ?

image
still not the same width

**Comment by Achintya-Chatterjee** [comment]
> > > * can we make the suggestion box width the same as the input box?
> > 
> > 
> > @yesyash can you see the video once again ?
> 
> image still not the same width

@vinit717 fixing it.

**Comment by Achintya-Chatterjee** [comment]
> > > > * can we make the suggestion box width the same as the input box?
> > > 
> > > 
> > > @yesyash can you see the video once again ?
> > 
> > 
> > image still not the same width
> 
> @vinit717 fixing it.

@vinit717 @yesyash it has been fixed

**Comment by vinit717** [comment]
Please increase the test coverage

**Comment by Achintya-Chatterjee** [comment]
> Please increase the test coverage line 99 and 431

@vinit717 will break the component, so I'm considering tackling this in the next PR.

**Review by vinit717 — APPROVED** [review]

**Review by yesyash — APPROVED** [review]

**Pull request merged**